src/components/molecules/FolderNavigatorFragment.tsx

```diff
@@ -50,6 +50,13 @@ const FolderListFragment = ({
     )
   }, [folderPathnames, storageId, sideNavOpenedItemSet])
 
+  const getFolderNoteCount = (folderPathname: string): number =>
```

Comment:
We cannot use this method everywhere, this will cause each folder nav item component to iterate every note for every change. But populated folders in our hook state are having `folderIdSet` props. This should be useful to show number of notes in a folder.